configure.ac

```diff
@@ -117,13 +117,23 @@ m4_ifdef([PKG_INSTALLDIR], [PKG_INSTALLDIR],
         [], [with_pkgconfigdir='$(libdir)/pkgconfig'])
     AC_SUBST([pkgconfigdir], [${with_pkgconfigdir}])])
 
+if echo $CC | grep -q gcc; then
```

Comment:
I didn't merge this check to master, because it can break build on some exotic OS/compiler/etc.